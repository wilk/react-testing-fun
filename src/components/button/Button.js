import React from 'react'
import Paragraph from "../paragraph/Paragraph"

let changingText = 'meh'
const onClick = () => {
  changingText = 'CHANGED'
}

export default () => {
  return (
    <button onClick={() => onClick()}>
      {changingText}
      <Paragraph/>
    </button>
  )
}