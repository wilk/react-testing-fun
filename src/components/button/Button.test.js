import React from 'react';
import { shallow } from 'enzyme';
import Button from './Button';

it('renders Button without Paragraph', () => {
  const wrapper = shallow(<Button />);
  expect(wrapper).toMatchSnapshot();
});