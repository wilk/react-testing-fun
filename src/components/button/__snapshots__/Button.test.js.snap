// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button without Paragraph 1`] = `
<button
  onClick={[Function]}
>
  meh
  <Component />
</button>
`;
